utils/build.ps1

```diff
@@ -1019,14 +1050,14 @@ function Build-CMakeProject {
       }
     }
 
-    if ($UseMSVCCompilers.Contains("C")) {
+    if ($C -eq "UseMSVC") {
```

Comment:
Why the string compares here?

Reply:
We could write `[Compilers]::UseMSVC` instead, but so far I was sticking to strings, because that's how we deal with `Platform` values so far.

Reply:
Added in https://github.com/swiftlang/swift/pull/77875